### DISCUSSION TITLE
After you create a process in one request, later requests sometimes will pending

### DISCUSSION THREAD

**Discussion by munhou** [fastapi/fastapi#8623] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import FastAPI
import multiprocessing
import time
import uvicorn

app = FastAPI()
def sleep():
    time.sleep(30)


@app.get("/")
def home():
    return {"message": "Hello World"}


@app.get("/create")
def read_root():
    t = multiprocessing.Process(target=sleep)
    t.start()
    return {"message": "Hello"}


if __name__ == "__main__":
    uvicorn.run("main:app", host="0.0.0.0", port=5000, log_level="info")
```

### Description

I think it's a very simple question, but I can't understand his phenomenon，After i create a process in one request, later requests sometimes will pending，Until the end of the previous process

### Operating System

Linux

### FastAPI Version

0.68.1

### Python Version

3.8.12

**Comment by iudeen** [comment]
Do you have enough CPU cores/threads to handle new process?

**Comment by yinziyan1206** [comment]
I guess fastapi create async task for every request. And you finish the request and return the response but the process is running. So it comes out.

**Comment by munhou** [comment]
> 

I have enough CPU cores，I work on a 12-core server，The frequency of blocking is not high, but he will appear。Is it possible that the request was routed to the process I created？

**Comment by munhou** [comment]
After requesting the create interface, wait about 10 seconds, subsequent requests will pendig

**Comment by YuriiMotov** [comment]
Works fine on my machine. At least with current version of FastAPI
